packages/x-components/src/x-modules/ai/store/actions/fetch-and-save-ai-suggestions-search.action.ts

```diff
@@ -0,0 +1,14 @@
+import type { AiXStoreModule } from '../types'
+
+export const fetchAndSaveAiSuggestionsSearch: AiXStoreModule['actions']['fetchAndSaveAiSuggestionsSearch'] =
```

Comment:
Try to use `createFetchAndSaveActions` helper to create this action

Reply:
that function relies on a `StatusState` adding a `status`, but how we deal with 2 different requests with just one 'status' flag ?